packages/common/index.ts

```diff
@@ -238,6 +239,47 @@ export type PhoneNotifBody = {
 // Office Hours Response Types
 export type GetProfileResponse = User;
 
+export class KhouryDataParams {
+  @IsString()
+  username!: string;
+  
+  @IsString()
+  email!: string;
+  
+  @IsString()
+  first_name!: string;
+  
+  @IsString()
+  last_name!: string;
+  
+  @IsOptional()
+  @IsString()
+  photo_url!: string;
+  
+  @IsOptional()
+  @IsDefined()  // TODO: use ValidateNested instead, for some reason it's crunked
+  courses!: KouryAdminCourse[];
+  
+  @IsOptional()
+  @IsDefined()  // TODO: use ValidateNested instead, for some reason it's crunked
```

Comment:
what's crunked?

Reply:
When I try to use the `@ValidateNested` decorator it can't validate the nested `KhouryAdminCourse` class